test/test_session_params.cpp

```diff
@@ -52,6 +56,13 @@ namespace
 		g_storage_constructor_invoked = true;
 		return dht_default_storage_constructor(settings);
 	}
+
+	sha1_hash to_hash(char const* s)
```

Comment:
I believe this utility function should already be available. it's possible though that it has been promoted to the general test library, in which case we can coalesce them later